Docker/validate_sc2.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+
+"""Validate SC2."""
+
+import argparse
+import json
+
+import pandas as pd
+
+
+def main(submission, entity_type, results="results.json"):
+    """Validate submission and write results to JSON.
+
+    Args:
+        submission: input file
+        entity: Synapse entity type
+        results: output file
+    """
+
+    invalid_reasons = []
+
+    if not submission.endswith(".csv"):
+        invalid_reasons.append(
+            "Prediction file must be a two-column CSV file.")
+    else:
+        predictions = pd.read_csv(submission, sep=",")
+
+        header = list(predictions.columns)
+        try:
+            assert header == ["lab_id", "responder"]
+        except AssertionError:
+            invalid_reasons.append("Column names must be: lab_id,responder.")
+
+        responses = set(predictions.responder)
+        try:
+            assert responses == {0, 1}
+        except AssertionError:
+            invalid_reasons.append(
+                " Responder values must be a Boolean (0 or 1).")
+
```

Comment:
Will probably want to make sure all labIds are accounted for.

Reply:
Oh right... that's what the goldstandard is for lol